### ISSUE TITLE
Reduce line-height on titles in EDS

### ISSUE THREAD

**Issue by emilsomm** [equinor/design-system#733]
**Is your feature request related to a problem? Please describe.**
In Figma the EDS typographical styles for titles have a to large line height. They are set with 1.5 ratio to the font size. This is too much for titles, which are generally more compact in expression compared to body text where this ratio is normal, and recommended. 

**Describe the solution you'd like**
For titles I would suggest a ratio of 1.1 which would achieve readability and more compactness. In the example below, I also suggest a `h1,h2,...{margin-bottom: 1em}` and `p:last-child { margin-bottom: 2em}` to achieve spacing according to [difi](https://uu.difi.no/krav-og-regelverk/webdirektivet-og-wcag-21/wcag-21-standarden/1412-tekstavstand-niva-aa) recommendations.

**Additional context**
I have made some examples for comparison and to show the context 

![figma art board](https://user-images.githubusercontent.com/20213342/96974781-fb55c500-1519-11eb-9f54-9a25a99eb7ef.png)

**Figma link:** https://www.figma.com/file/cydecneJ8Kc4ErBS3GsoXt/lineheight?node-id=0%3A1

**Comment by vnys** [comment]
Great, thank you 😊 We do have typography on our list this autumn, we’re well aware that there’s room for improvement in this area, so we’ll definitely take your proposal into consideration.

**Comment by mimarz** [comment]
blocked by #918

**Comment by BirteThornquist** [comment]
The suggested change of line height for Heading1 is 1,1 of font size. I think this is too little. I agree that 1,5 in large headings are too much, though. My preference would be 1,3. In addition to the example made in this issue I have made some examples [here](https://www.figma.com/file/dCtGxSjiufV5C6YcXSs3MD/?node-id=447%3A111423). What do you say @lucasveil and @vnys ?

**Comment by lucasveil** [comment]
I didn't find any Difi recommendation on the subject (specific to line-height) and according to [Design System](https://www.designsystems.com/typography-guides/#:~:text=A%20line%2Dheight%20that%20is,to%20or%20divide%20equally%20into.) is the 1.5 ratio the "rule of thumb". @vnys would be nice to have your feedback on this?

**Comment by vnys** [comment]
Damn, I wrote a lot and it disappeared before I managed to hit comment 🤦🏻‍♂️ I need a draft function. Short version, the [success criterion 1.4.12 AA](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html) says 1.5em for line-height, but even Deque use 1.35em for headings on their the web site – so I think the most important thing here is to focus on body copy.

**Comment by BirteThornquist** [comment]
Agreed @vnys. I'd say we go ahead with 1,3. That reads well.

**Comment by mimarz** [comment]
Just to throw in there, inspecting https://www.uutilsynet.no/wcag-standarden/sidetittel-navigasjon/221 shows using `line-height: 1,3`  also (overriding a default `1,2` 🤔)

![image](https://user-images.githubusercontent.com/1070981/114983943-7d072980-9e91-11eb-902a-beca8b2a3fd4.png)

**Comment by BirteThornquist** [comment]
hm, interesting. Maybe 1,2 was too close the..

**Comment by vnys** [comment]
On a side note, I added some examples of type scales in that same page in Figma – that’s also something we’ve had in the backlog for a while to improve – so I thought I’d set up some examples to understand what other projects do.

**Comment by BirteThornquist** [comment]
After a discussion we decided to change the line height from 1,5 to 1,25 for headings H1 - H3.  We have tested different line heights and looked into how others are doing it. For now we keep the font sizes but want to look into an adjustment of the scale later on. We want to gather some more feedback from our users and are planning a Typograpics theme for one of our meet the designer meeting.  We should also change the naming from H1 etc to instead referanse the size of the font and not the semantic.
